Misc/NEWS.d/next/IDLE/2018-12-24-17-17-21.bpo-34055.zDC3Va.rst

```diff
@@ -0,0 +1 @@
+IDLE Shell: Fix printing smart indent on every line
```

Comment:
This patch only cleans the code and should have no visible behavior change.  I will change this when I add an idlelib.NEWS entry.